Automattic/newspack-plugin
# feat(ras-acc): add faq section to otp email

### All Submissions:






### Changes proposed in this Pull Request:

**PR 3 of 4**



This PR adds onto the work in https://github.com/Automattic/newspack-plugin/pull/3032 and https://github.com/Automattic/newspack-plugin/pull/3033 by adding an FAQ section to the otp email:

![Screenshot 2024-04-05 at 13 16 04](https://github.com/Automattic/newspack-plugin/assets/17905991/1758cdb3-010f-4a99-9140-efcbba0c09e6)

### How to test the changes in this Pull Request:

1. With these changes checked out, go to Newspack > Engagment > Show Advanced Settings and select the one-time password email under Transactional Email Content section
2. Trash the post (so that we can force our updated template to be used on next otp sign in attempt)
3. As a logged out user, select the sign in link and attempt to login via email to trigger an otp email
4. Check the otp email and confirm the FAQ section is present and the link to set a new password works as expected

### Other information:
